app/src/main/java/com/framgia/soundclound/screen/home/HomeViewModel.java

```diff
@@ -1,21 +1,30 @@
 package com.framgia.soundclound.screen.home;
 
+import android.content.Context;
+import android.content.Intent;
 import android.databinding.BaseObservable;
 import android.databinding.Bindable;
+import android.support.annotation.IntDef;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports